go/vt/tabletmanager/state_change.go

```diff
@@ -51,12 +51,18 @@ func (agent *ActionAgent) loadBlacklistRules(tablet *topodatapb.Tablet, blacklis
 		if err != nil {
 			return err
 		}
-		log.Infof("Blacklisting tables %v", strings.Join(tables, ", "))
-		qr := tabletserver.NewQueryRule("enforce blacklisted tables", "blacklisted_table", tabletserver.QRFailRetry)
-		for _, t := range tables {
-			qr.AddTableCond(t)
+
+		// if the wildcards resolve into real tables, blacklist them.
+		// (if there are no tables here, the rule would blacklist
```

Comment:
Heh, maybe this comment is better to rewrite as "Verify that at least one table matches the wildcards, so that we don't add a rule to blacklist all tables"